src/nvim/eval.c

```diff
@@ -15219,10 +15219,13 @@ static void f_setline(typval_T *argvars, typval_T *rettv)
 /*
  * Used by "setqflist()" and "setloclist()" functions
  */
-static void set_qf_ll_list(win_T *wp, typval_T *list_arg, typval_T *action_arg, typval_T *rettv)
+static void set_qf_ll_list(win_T *wp, typval_T *list_arg, typval_T *action_arg,
+                           typval_T *title_arg, typval_T *rettv)
+                           FUNC_ATTR_NONNULL_ARG(2, 3, 4, 5)
```

Comment:
This is too shifted. Function attributes are indented as function definition start + shiftwidth and not aligned.